### ISSUE TITLE
validations are a bottleneck

### ISSUE THREAD

**Issue by danspecht** [hotosm/osm-tasking-manager2#568]
Lots of projects are mapped quickly, but validated  slowly. This could be because 
(A) beginners don't feel qualified to pass judgement
(B) people don't like to pass judgement
(C)  doing original work is more fun than reviewing someone else's work.

I have a couple suggestions for encouraging validation.

1.  Include instructions for validation on the instructions tab.  
   Because the instructions tab only has mapping instructions, readers may think that validation is for someone else to do.

2.  Include validation statistics on the stats tab. 
  Because the stats tab only has statistics for tiles completed, mappers may think that validating tiles is not essential. Also, these statistics give the mapper, but not the validator, a psychological reward. I've been validating a lot of tiles -- sometimes I seem to be doing most of the validations on a project -- and even though seeing the number by your name increment isn't the biggest thrill in the world, I have to admit that I miss it..

**Comment by ethan-nelson** [comment]
I agree that validation stats would be useful to have on the stats tab. Right now, the activity is essentially unacknowledged...and some people specifically work on validation (which I'd argue is just as important as the initial mapping). I can try and take the task of working on adding validation stats. Would love more input on whether the validation instructions should task-specifiable or rather something that repeats the items to be mapped and adds "Check that these have been mapped:".

**Comment by ethan-nelson** [comment]
Perhaps on the user page, also, we could distinguish how a user contributed to each task...X tiles mapped, Y tiles validated.

**Comment by pierzen** [comment]
Analyzing Ebola contribution, I realize that we have to take account both of the Duration + Intensity of contribution. Total nb of changesets would complete the info about contributor experience since old accounts with a few changesets do not show intensive mapping experience.

**Comment by ethan-nelson** [comment]
@pierzen I have been in the process of providing the information directly on the user page regarding changesets and time with OSM (https://github.com/hotosm/osm-tasking-manager2/pull/538)...I'm just needing to integrate i18n.

**Comment by pierzen** [comment]
**Validation Panel priviledges**
To better control the validation process, Access to this Validation panel that I propose could be reserved to experienced contributors. A new User privilege Validator could be added. Both Administrators and Project managers would have de facto the Validator priviledge.

**Comment by danspecht** [comment]
Ethan,

When I validate, I just verify that what was requested in the instructions
was done.  If it's almost done I finish it myself.  If that kind of
validation is OK, than I think generic instructions will be fine.

Dan

On Thu, Apr 9, 2015 at 7:57 AM, Ethan <notifications@github.com> wrote:

> I agree that validation stats would be useful to have on the stats tab.
> Right now, the activity is essentially unacknowledged...and some people
> specifically work on validation (which I'd argue is just as important as
> the initial mapping). I can try and take the task of working on adding
> validation stats. Would love more input on whether the validation
> instructions should task-specifiable or rather something that repeats the
> items to be mapped and adds "Check that these have been mapped:".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hotosm/osm-tasking-manager2/issues/568#issuecomment-91255235>
> .
>



-- 
Dan

**Comment by CloCkWeRX** [comment]
I would be keen to drop the barrier to entry as much as possible - #626 for example would complement 'show validation instructions when validating' really well I think.

**Comment by pierzen** [comment]
We could have various strategies to validate a Job. For exemple, a list of users by experience + Stats (creation account, nb changesets, nb HOT tasks, nb HOT tasks invalidations). This invalidation would need to be documented, since some validators do invalidation without good motives.

After the Nepal earthquake response and the no of new contributors, we should have a Validation group to think at strategies to validate. This could also include other tools to validate geometry, tagging schema semantic, road classification etc.

**Comment by bgirardot** [comment]
This issue should probably be merged with issue #545 "Statistics on completed squares for validators"

**Comment by danspecht** [comment]
Github,

Do you think that there should be a third option for poor imagery? If I
declare it invalid, someone may try to correct it, wasting time. However if
I declare it valid, no one will bother to update it when better imagery
becomes available.

-- 
Dan

**Comment by bgirardot-np** [comment]
@danspecht Tasking Manager 3 will have an option for bad imagery
